moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -489,7 +491,8 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
   }
 
   std::string retimeTrajectory(const std::string& ref_state_str, const std::string& traj_str,
-                               double velocity_scaling_factor)
+                               double velocity_scaling_factor, double acceleration_scaling_factor,
+                               std::string algorithm)
```

Comment:
This has caused a [Travis breakage on master](https://travis-ci.org/ros-planning/moveit/jobs/563596256), it looks like it got through because this was committed to melodic first, then cherry-picked?

Reply:
Fix: https://github.com/ros-planning/moveit/pull/1589

Reply:
Thank you. Sorry for the inconvenience.